notebooks/BTA_array_producer.ipynb

```diff
@@ -0,0 +1,2322 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # @nb.njit # sometime has bug... strange!
</pre><p>out of curiosity: what kind of bug is it?</p><p>Does this bug only occur in this line or in line 4 of the HLT section and line 73 of the Hadrons section as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cms-btv-pog/BTVNanoCommissioning/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I think it's the issue when using numba in the matching process, indeed he didn't use numba in the code to generate the `fromQGP` info. I didn't use the same method, no numba bug.

Reply:
Ok thanks for clarifying